### PR TITLE
wpewebkit: add accessibility support

### PR DESCRIPTION
**Pull request by psaavedra**

### CODE REVIEW

**Comment by philn** [comment]
Is a11y supported in 2.24 actually?

**Comment by philn** [comment]
I don't think it makes sense to enable this now, a11y won't be supported in wpewebkit until the 2.26 series.

**Comment by psaavedra** [comment]
Aham. I see, in my mind wpewebkit_trunk.bb was using the latest trunk revision. In this case, yes, this is not needed for until a new recipe for 2.26.

File: recipes-browser/wpewebkit/wpewebkit.inc [hunk 1]
```diff
@@ -22,6 +22,7 @@ TOOLCHAIN = "gcc"
 PACKAGECONFIG ??= "fetchapi indexeddb mediasource video webaudio webcrypto woff2 gst_gl remote-inspector openjpeg"
 
 # WPE features
+PACKAGECONFIG[accessibility] = "-DENABLE_ACCESSIBILITY=ON,-DENABLE_ACCESSIBILITY=OFF,atk"
```
Comment (philn) [inline recipes-browser/wpewebkit/wpewebkit.inc]: at-spi2-core isn't needed there? I had to have it when i tried to enable a11y with a trunk snapshot.

**Comment by psaavedra** [comment]
> Aham. I see, in my mind wpewebkit_trunk.bb was using the latest trunk revision. In this case, yes, this is not needed for until a new recipe for 2.26.

I added it in the bbappend of  meta-perf-browser (https://gitlab.com/saavedra.pablo/meta-perf-browser/commit/3093e01b0881c5d8bebe5dbbf9881a2adcef4b0c)  . The one used to run the CI against trunk so not needed right now in this layer. We can way for:

* A bump-up of the trunk recipe
* The release of 2.26

File: recipes-browser/wpewebkit/wpewebkit.inc (hunk 1 above)
Comment (psaavedra) [reply recipes-browser/wpewebkit/wpewebkit.inc]: done

**Pull request merged**